src/main/java/io/github/cottonmc/cotton/gui/widget/WAbstractSlider.java

```diff
@@ -30,9 +30,17 @@
 	 */
 	private static final int DRAGGING_FINISHED_RATE_LIMIT_FOR_SCROLLING = 10;
 
-	protected final int min, max;
+	protected int min, max;
 	protected final Axis axis;
 
+	/**
+	 * Setters for min and max
+	 */
+	public void setMinAndMax(int min, int max) {
+		this.min = min;
+		this.max = max;
+	}
+
```

Comment:
I think it would be better to add separate `setMinValue` and `setMaxValue` setters below the getters.
https://github.com/CottonMC/LibGui/blob/e511933cb169218373b70084a133b50f28175560/src/main/java/io/github/cottonmc/cotton/gui/widget/WAbstractSlider.java#L213-L219

Reply:
Fixed.